multimodal/setup.py

```diff
@@ -33,8 +33,8 @@
     "evaluate>=0.2.2,<0.4.0",
     "accelerate>=0.9,<0.17",
     "timm>=0.9.2,<0.10.0",
-    "torch>=1.9,<1.14",
-    "torchvision<0.15.0",
+    "torch>=1.13,<2.1",
```

Comment:
Noticed torch requirements appear in multiple modules. Not sure if it's doable to synchronize them into once place especially when torch for tabular is in extra dependency. @Innixma Any idea?